src/makeRobot.js

```diff
@@ -38,7 +38,74 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name},`
+      + ` chip version: ${this.version},`
+      + ` wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: `
+        + `x=${this.coords.x}, `
+        + `y=${this.coords.y}`;
+    },
+
+    goRight(steps = 1) {
+      if (steps < 0 || typeof steps !== 'number') {
+        return this;
+      }
+
+      this.coords.x += steps;
+
+      return this;
+    },
```

Comment:
```suggestion
    goRight(steps = 1) {
      if (steps >= 0) {
        this.coords.x += steps;
      }

      return this;
    },
```
If you change the conditions you can make your code shorter